### ISSUE TITLE
Documentation is not available in the docs github

### ISSUE THREAD

**Issue by noloader** [coreruleset/owasp-crs-documentation#2]
Hi Everyone,

My apologies for this issue. I can't find the documentation in a ready to read state.

I started at https://github.com/SpiderLabs/owasp-modsecurity-crs/issues/656. https://github.com/SpiderLabs/owasp-modsecurity-crs/issues/656 took me to https://github.com/coreruleset/coreruleset/.

https://github.com/coreruleset/coreruleset/ has a docs/ folder at https://github.com/SpiderLabs/owasp-modsecurity-crs/tree/v3.3/dev/docs. docs/ took me to https://github.com/coreruleset/owasp-crs-documentation/tree/1b5ea0c88c33.

I'm at https://github.com/coreruleset/owasp-crs-documentation/tree/1b5ea0c88c33, but I don't see any documentation (yet). There is nothing to read.

There's also a note that the latest documentation is at https://www.modsecurity.org/CRS/Documentation/. https://www.modsecurity.org/CRS/Documentation/ does not have any docs to actually read.

A mailing list is equally difficult to find... [OWASP ModSecurity Core Rule Set mailing list](https://lists.owasp.org/mailman/listinfo/owasp-modsecurity-core-rule-set) results in a 404. Following the link to [Find it Here](https://groups.google.com/a/owasp.org/forum/#!overview) takes me to Google Groups home page, without taking me to the actual group.

This is a usability nightmare...

Let me state the obvious. Please provide a PDF or DOC so we can read the docs.

**Comment by gustavokuhl** [comment]
Hi,

I'm in the ModSec community for a long time, but it is what I found.

You can buy the following book about ModSecurity (written by the creator of ModSecurity), or you can just read the free part of the same book:
https://www.feistyduck.com/books/modsecurity-handbook/

**Comment by dune73** [comment]
@noloader: You are absolutely correct and we have been aware of this problem for much too long. I'm writing you from the CRS developer summit and we just finished our first workshop about the new documentation that we are rebuilding from scratch. See https://github.com/coreruleset/coreruleset/wiki/DevRetreat21UpdateDocumentation for coverage of our plans.
